### PR TITLE
YapDatabase: Fix compilation warnings.

### PR DESCRIPTION
**Pull request by Shachlan**

### CODE REVIEW

**Comment by Shachlan** [comment]
@CodeBaseCamp

File: YapDatabase/Extensions/SecondaryIndex/YapDatabaseSecondaryIndexTransaction.m [hunk 1]
```diff
@@ -1573,7 +1573,8 @@ - (id)performAggregateQuery:(YapDatabaseQuery *)query
 	
 	[databaseTransaction _enumerateRowidsForKeys:keys
 	                                inCollection:collection
-	                         unorderedUsingBlock:^(NSUInteger keyIndex, int64_t rowid, BOOL *stop)
+	                         unorderedUsingBlock:^(NSUInteger keyIndex, int64_t rowid,
+                                                 BOOL __unused *stop)
```
Comment (CodeBaseCamp) [inline YapDatabase/Extensions/SecondaryIndex/YapDatabaseSecondaryIndexTransaction.m]: Indentation

File: YapDatabase/Utilities/YapDatabaseQuery.m [hunk 2]
```diff
@@ -247,7 +247,8 @@ + (instancetype)queryWithAggregateFunction:(NSString *)inAggregateFunction
 	
 	NSMutableDictionary *paramIndexToArrayCountMap = [NSMutableDictionary dictionary];
 	
-	[inQueryParameters enumerateObjectsUsingBlock:^(id param, NSUInteger paramIndex, BOOL *stop) {
+	[inQueryParameters enumerateObjectsUsingBlock:^(id param, NSUInteger paramIndex,
+                                                  BOOL __unused *stop) {
```
Comment (CodeBaseCamp) [inline YapDatabase/Utilities/YapDatabaseQuery.m]: same

File: YapDatabase/Extensions/SecondaryIndex/YapDatabaseSecondaryIndexTransaction.m (hunk 1 above)
Comment (Shachlan) [inline YapDatabase/Extensions/SecondaryIndex/YapDatabaseSecondaryIndexTransaction.m]: done.

File: YapDatabase/Utilities/YapDatabaseQuery.m (hunk 2 above)
Comment (Shachlan) [inline YapDatabase/Utilities/YapDatabaseQuery.m]: done.

**Comment by Shachlan** [comment]
Finished fixing.

**Pull request merged**